src/patternfly/components/Nav/nav.scss

```diff
@@ -252,16 +252,11 @@
     margin-block-end: var(--#{$nav}__button--RowGap--row-offset);
   }
 
-  &.pf-m-expanded:is(:not(:only-child, :last-child)) {
-    + .#{$nav}__item > .#{$nav}__link::before {
-      inset-block-start: calc(var(--#{$nav}__item--RowGap) * -1 - var(--#{$nav}__list--RowGap));
-    }
+  &:is(:only-child, :last-child) {
```

Comment:
I think with `:last-child` it means the last item in the subnav is still getting cut off. I think that can safely be removed. WDYT?